barretenberg/cpp/src/barretenberg/translator_vm/translator_flavor.hpp

```diff
@@ -304,7 +291,8 @@ class TranslatorFlavor {
                                WireToBeShiftedEntities<DataType>::get_all());
         };
 
-        // Used when computing commitments to wires + ordered range constraints during proof consrtuction
+        // Used when computing commitments to wires + ordered range constraints during proof
```

Comment:
typo here and maybe an incorrect line break here an below?

Reply:
I hate when the line breaks happen randomly